src/v3/package.json

```diff
@@ -125,7 +125,7 @@
     "stylelint-use-logical-spec": "^5.0.0",
     "svgo": "^2.3.0",
     "terser-webpack-plugin": "^5.2.4",
-    "testcafe": "^1.20.1",
+    "testcafe": "^3.3.0",
```

Comment:
🎉 

Wonder if this can be done for the root version of `testcafe` as well?

Reply:
@shawyu-okta im having issues in bacon with this upgrade.  I might have to revert this back to the original

Reply:
testcafe version was reverted back to original due to bugs and performance issues